packages/buttons/demo/anchor.stories.mdx

```diff
@@ -11,7 +11,14 @@ import { Anchor } from '@zendeskgarden/react-buttons';
 
 <Canvas>
   <Story
-    args={{ children: 'Text' }}
+    args={{ children: 'Text', href: '#', externalIconTextLabel: 'Label' }}
+    argTypes={{
+      href: {
+        table: {
+          disable: true
+        }
+      }
+    }}
```

Comment:
By convention, we limit this to Story controls (only `disable` knocking-out when necessary). The `href` can be removed here and applied directly within the story below as `<Anchor href="#" {...args} />`